### PR TITLE
Back port from pull request #2052 for 74X

### PR DESCRIPTION
**Pull request by skeshri**
Pede Revision 148: /tags/V04-03-03 "with Function not decreasing Fix"
cherry picked from commit 08d7a2e920e7c543eb59984221736e3c43098aaa

### CODE REVIEW

File: millepede.spec [hunk 1]
```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
```
Comment (ghost) [inline millepede.spec]: @skeshri you need to solve the merge conflict and test it yourself before creating a new PR. (also, the difference between the two version is 2 numbers of the version, you can easily modify them without merging or cherry-picking...)

**Comment by ghost** [comment]
@cmsbuild please test

**Comment by skeshri** [comment]
Hi @degano ! Could you please tell me why test is pending for this pull request? Is there anything wrong with this pull request?

**Comment by ghost** [comment]
@cmsbuild please test

**Comment by smuzaffar** [comment]
@degano, I think this should be done for IB/CMSSW_7_4_X/root6 (which is production branch for 74x)

**Comment by ghost** [comment]
Right, @skeshri please recreate this PR for the production branch mentioned above.

**Comment by skeshri** [comment]
Thanks @degano  and @smuzaffar  !
I will recreate pull request for the branch IB/CMSSW_7_4_X/root6 again.

**Pull request closed without merging**